pontoon/api/tests/test_views.py

```diff
@@ -0,0 +1,615 @@
+import pytest
+
+from rest_framework.test import APIClient
+
+from pontoon.base.models.project import Project
+from pontoon.base.models.resource import Resource
+from pontoon.base.models.translation_memory import TranslationMemoryEntry
+from pontoon.terminology.models import Term, TermTranslation
+from pontoon.test.factories import (
+    LocaleFactory,
+    ProjectFactory,
+    ResourceFactory,
+    TranslatedResourceFactory,
+)
+
+
+@pytest.mark.django_db
+def test_locale():
+    locale_a = LocaleFactory(
+        code="kg",
+        name="Klingon",
+    )
+    response = APIClient().get(
+        f"/api/v2/locales/{locale_a.code}/", HTTP_ACCEPT="application/json"
+    )
+    assert response.status_code == 200
+
+    assert response.data == {
+        "code": "kg",
+        "name": "Klingon",
+        "direction": "ltr",
+        "population": 0,
+        "cldr_plurals": "",
+        "plural_rule": "",
+        "script": "Latin",
+        "google_translate_code": "",
+        "ms_terminology_code": "",
+        "ms_translator_code": "",
+        "systran_translate_code": "",
+        "team_description": "",
+        "total_strings": 0,
+        "approved_strings": 0,
+        "pretranslated_strings": 0,
+        "strings_with_warnings": 0,
+        "strings_with_errors": 0,
+        "missing_strings": 0,
+        "unreviewed_strings": 0,
+        "complete": True,
+        "projects": ["terminology"],
+    }
+
+
+@pytest.mark.django_db
+def test_locale_stats():
```

Comment:
Wouldn't it be simpler to join this test with `test_locale()`? 
And same for `test_project_stats()` and `test_project()`?

Also, why don't we check the number of DB queries in all tests?